Makefile

```diff
@@ -8,16 +8,10 @@ FS_MODULES=/usr/lib/freeswitch/mod
 SHELL := /bin/bash
 PROC?=$(shell uname -m)
 
-CC=gcc 
 CFLAGS=-fPIC -O3 -fomit-frame-pointer -fno-exceptions -Wall -std=c99 -pedantic
-ifeq (${PROC},x86_64)
```

Comment:
I don't know what maybe the best approach.

What seems reasonable to me is to be able to build without external flags on the same *nix platform supported freeswitch && bcg729 , with an exception of x86_64/x86 because is my opinion that the cross compilation between those two is (unfortunately, still) common.

what do you think?